docs/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
What's this license for does it need to be included? It makes it feel a bit like `xhr-mock` is copyrighted to `CloudCannon`.

Reply:
It's a bit difficult to find information on whether or not we can get rid of the CloudCannon license though its an MIT licensed template. One option we could consider is adding your name to the license, since this project is also under MIT, specifying the api doc template is CloudCannon and the API is yours. I could attempt to contact the template creator to ask what he is okay with.